travis_ci/install_docs.bash

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -e
+
+conda create -y -n test python=3.7 sphinx mock sphinx_rtd_theme m2r
+source $HOME/miniconda/etc/profile.d/conda.sh
+conda activate test
+
+if [[ "$TRAVIS_PULL_REQUEST" == "false" ]]; then
+  export DOCS_VERSION="$TRAVIS_BRANCH"
+elif [ -n "$TRAVIS_TAG" ]; then
+  # this is a tag build
+  export DOCS_VERSION="$TRAVIS_TAG"
+else
+  export DOCS_VERSION=7.0
```

Comment:
Same question about parsing the MPAS-Model version from the README.md or elsewhere.